docs/EN_US/ECLStandardLibraryReference/SLR-Mods/YearWeekNumFromDate.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect1 PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<sect1 id="YearWeekNumFromDate">
+  <title>YearWeekNumFromDate</title>
+
+  <para><emphasis role="bold">STD.Date.YearWeekNumFromDate<indexterm>
+      <primary>STD.YearWeekNumFromDate</primary>
+    </indexterm><indexterm>
+      <primary>YearWeekNumFromDate</primary>
+    </indexterm>(</emphasis> <emphasis role="bold">date,
+  startingDayOfWeek)</emphasis></para>
+
+  <informaltable colsep="1" frame="all" rowsep="1">
+    <tgroup cols="2">
+      <colspec colwidth="80.50pt" />
+
+      <colspec />
+
+      <tbody>
+        <row>
+          <entry><emphasis>date</emphasis></entry>
+
+          <entry>The date (in Date_t format) for which to compute the week
+          number; must be greater than or equal to referenceDate.</entry>
```

Comment:
Remove "must be greater than or equal to referenceDate" -- that comment actually applies to the SHARED function that this one calls.